.eslintrc.js

```diff
@@ -33,15 +33,6 @@ module.exports = {
     'import/no-namespace': 'off',
     'import/no-named-export': 'off',
     'no-unused-vars': 'off',
-    'prettier/prettier': [
-      'error',
-      {
-        arrowParens: 'always',
-        printWidth: 100,
-        singleQuote: true,
-        trailingComma: 'none',
-        plugins: ['prettier-plugin-package']
-      }
-    ]
+    'prettier/prettier': 'error'
```

Comment:
I have moved prettier config to a dedicated file that is shared in all places and provides better IDE support. Relevant change is probably that the `printWidth` in README files is now 100 characters as well. If this is not desired, I can add an override for markdown files to set it to 80 again.

Reply:
Yeah please do revert this. This configuration keeps prettier and ESLint from fighting, and requires only a single editor extension. There's a lot of debate about this, but reverting this change is going to cause more trouble than it's worth.

Reply:
And if we just double the config from the .prettierrc file in the .eslintrc.js file? In any case, I would really love to get rid of the manual prettier `--print-width` params in favour of central config. As the eslintrc is central as well, that would be fine with me.